libp2p/crypto/ed25519/ed25519.nim

```diff
@@ -13,7 +13,8 @@
 
 {.push raises: Defect.}
 
-import constants, bearssl
+import bearssl/rand
```

Comment:
should export `rand` here too since there's a public symbol that references it - this way, the tests and examples no longer explicitly have to import `bearssl` and it can be removed completely, making the lib easier to use